2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2039 @@
+process requires the installation of new funding instruments.
+Traditionally, research grants are limited to rather short time frames
+and support personnel, material, hardware, and to a limited degree also
+travel and research visits. Creating a research software community
+however requires funding for community and training events as well as
+“virtual hardware” such as webspace, versioning systems, task-managing
+systems, and compute cycles. These demands can hardly be met without
+third-party funding (Kuchinke et al. 2016; Chang, Mills, and Newhouse
+2007; Aartsen et al. 2018; Gabella, Durinx, and Appel 2018).
+
+Recommendation: Creation of Adequate Funding Schemes
+----------------------------------------------------
+
+Funding is a crucial factor for sustaining research software. Currently
+available sources and instruments are not adequately shaped for the
+challenges and solutions outlined above. We recommend actions on the
+individual, organizational, and national level.
+
+Existing project-focused funding instruments on the local, national, and
+international level need to be complemented with funding instruments
+specifically designed for research software development and sustained
+research software maintenance to make research software a first class
+citizen in the research landscape. For example, software projects
+enhancing research and fulfilling the sustainability criteria detailed
+above may be entitled for sustained funding as long as they
+live up to the standards and remain a central component of the research
+landscape.
+
+Computing centers and supercomputing facilities for research need to
+receive earmarked resources for the support of sustainable software
+development. This funding is necessary to provide continuous integration
+services, a hardware portfolio for development, testing and benchmarking
+software, as well as personnel for training domain researchers in
+software design and the proper usage of the services.
+
+The creation and maintenance of training material for general research
+software engineering education and the software-specific documentation
+and tutorial creation needs to be reflected in funding opportunities.
+This can either happen by dedicating modules of research or software
+grants to providing support and the generation of training material, or
+by opening funding schemes focusing on interdisciplinary software
+development education. The latter may include research that looks at
+research software development as a process to analyze which measures,
+interactions, and team compositions make research software successful.
+Additionally, funding instruments fostering the formation of research
+software communities have to be established.
+
+Which Infrastructure is Needed to Sustain Research Software?
+============================================================
+
+> As the *hal9k* community grows, so does the need for infrastructure.
+> Kim and her team collaborate with 
```

Comment:
```suggestion
landscape. While some funding for such tools and software repositories is
emerging (e.g. the bio.tools catalogue of bioinformatics tools funded
as part of the European ELIXIR project, Ison, Rapacki, Ménager et al., 
2015), a standardized extension of such efforts to the RSE community as
a whole is necessary. However, as the experiences from ELIXIR
demonstrate, this is a non-trivial effort that requires significant dedicated
and long-term funding. 
```

From #6

Reply:
TODO Add reference to Overleaf.

Jon Ison, Kristoffer Rapacki, Hervé Ménager, Matúš Kalaš, Emil Rydza, 
Piotr Chmura, Christian Anthon, Niall Beard, Karel Berka, et al. 2015.
“Tools and data services registry: a community effort to document 
bioinformatics resources” *Nucleic Acids Research* 44 (January): D38-D47.

Reply:
Added with citekey ison2016tools